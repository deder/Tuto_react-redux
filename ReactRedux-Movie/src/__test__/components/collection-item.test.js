import CollectionItem from '../../components/collection-item';
import React from 'react';

describe("<CollectionItem />",()=>{
    it("CollectionItem avec image", () => {
        const onClick = jest.fn();
        const wrapper = mount(
            <CollectionItem onClick={onClick} image="test.png" title="title" description="description"  />
        );
        expect(wrapper).toMatchSnapshot();
    });
    it("CollectionItem Sans image", () => {
        const onClick = jest.fn();
        const wrapper = mount(
            <CollectionItem onClick={onClick} title="title" description="description"  />
        );
        expect(wrapper).toMatchSnapshot();
    });
})