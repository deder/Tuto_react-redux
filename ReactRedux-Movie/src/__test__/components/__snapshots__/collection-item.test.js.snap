// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CollectionItem /> CollectionItem Sans image 1`] = `
<li
  className="collection-item avatar"
  onClick={[Function]}
>
  <span
    className="title"
  >
    title
  </span>
  <p>
    description
  </p>
</li>
`;

exports[`<CollectionItem /> CollectionItem avec image 1`] = `
<li
  className="collection-item avatar"
  onClick={[Function]}
>
  <img
    alt="title"
    className="circle"
    src="test.png"
  />
  <span
    className="title"
  >
    title
  </span>
  <p>
    description
  </p>
</li>
`;
